src/components/tips/Tips.js

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import { Button } from "react-bootstrap";
+import { Link } from "react-router-dom";
+import "./tips.css";
+export default function Home() {
+  return (
+    <>
+      <div className="tips__container">
+        <h1>
+          <u>Tips to adopting Pets</u>
+          <br></br>
+          <br></br>
+        </h1>
```

Comment:
you can add the underline in h1 instead of using `<u>`
 
 ```
.tips__container h1{
  text-decoration: underline;
}
```

instead of using `<br>` you could use bootstrap5 [utility spacing](https://getbootstrap.com/docs/5.0/utilities/spacing/) for the margin bottom spacing

`<h1 className="mb-1>`

Reply:
Replaced <br> with bootstrap class but I can't do the first thing, it will underline everything.